docs/UserGuide.adoc

```diff
@@ -158,6 +195,23 @@ Selects the 2nd person in the address book.
 `select 1` +
 Selects the 1st person in the results of the `find` command.
 
+=== Rating a student : `rate`
+
+Rates the student identified by the index number used in the last person listing. +
+Format: `rate INDEX t/TECHNICAL_SKILLS_SCORE c/COMMUNICATION_SKILLS_SCORE p/PROBLEM_SOLVING_SKILLS_SCORE e/EXPERIENCE_SKILLS_SCORE`
+
+****
+* Rates the student at the specified `INDEX`. The index refers to the index number shown in the last person listing. The index *must be a positive integer* 1, 2, 3, ...
+* The scores should be a number in the range of 1 to 5 (inclusive).
+****
+
+Examples:
+
+* `rate 1 t/4.5 c/3 p/4 e/3.5` +
+Rates the 1st person to have technical skills score of 4.5, communication skills score of 3, problem solving skills of 4, experience score of 3.5.
```

Comment:
Since your description says `rates the student`, ensure that it is consistent throughout all the documentation.